### PR TITLE
[Auth] Add missing field to firebase claim in token result

### PR DESCRIPTION
**Pull request by sam-gc**
Fixes https://github.com/firebase/firebase-js-sdk/issues/6218

### CODE REVIEW

**Review by yuchenshi — APPROVED** [review]

**Review by lisajian — APPROVED** [review]

File: .changeset/cool-elephants-leave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/auth": patch
+---
+
+Add missing field to firebase claim in token result typing
```
Comment (egilmorez) [inline .changeset/cool-elephants-leave.md:5]: Should Firebase be capped here, or is it `firebase` a literal?

**Review by egilmorez — APPROVED** [review]
LG with possible nit thanks!

File: .changeset/cool-elephants-leave.md (hunk 1 above)
Comment (sam-gc) [reply .changeset/cool-elephants-leave.md:5]: Firebase literal, should I put it in backticks?

Comment (sam-gc) [reply .changeset/cool-elephants-leave.md]: Updated for backticks

**Review by egilmorez — APPROVED** [review]
Still LG!

**Review by lisajian — APPROVED** [review]

**Review by yuchenshi — APPROVED** [review]

**Review by hsubox76 — APPROVED** [review]

**Pull request merged**